src/styles/blocks/calendar.scss

```diff
@@ -1,47 +1,45 @@
 .calendar {
   display: flex;
+  margin: 0 auto;
```

Comment:
```suggestion
```
Better add styling to the parent block to center calendar. Blocks don't dictate their position themselves.

Reply:
please, fix ALL comments, including this one